website/docs/user_guide/create-feature-toggle.md

```diff
@@ -3,39 +3,141 @@ id: create_feature_toggle
+:::
 
-![Create a feature toggle](/img/create_feature_toggle_userIds.png)
+In the project that you want to create the toggle in, use the "new feature toggle" button and fill the form out with your desired configuration. Refer to the [feature toggle reference documentation](../reference/feature-toggles.mdx) for the full list of configuration options and explanations.
 
-## Summary {#summary}
 
-In this guide you created your first feature toggle and enabled it for everyone. In later guides we will learn how we can reduce the risk by enabling the toggle for a controlled set of users first.
+![](/img/create-toggle-new-toggle.png)
+
+### Step 2: Add a strategy {#step-2}
+
+:::tip API: Add a strategy
+
+Use the [API for adding a strategy to a feature toggle](../api/admin/feature-toggles-api-v2.md#add-strategy). You can find the configuration options for each strategy in the [activation strategy reference documentation](../user_guide/activation-strategies.md).
+
+:::
+
+Decide which environment you want to enable the toggle in. Select that environment and add an activation strategy. Different activation strategies will act differently as described in the [activation strategy documentation](../user_guide/activation-strategies.md). The configuration for each strategy differs accordingly. After selecting a strategy, complete the steps to configure it.
+
+![](/img/create-toggle-add-strategy.png)
+
+### Step 3: Enable the toggle {#step-3}
+
+:::tip API: Enable a toggle
+
+Use the [use the API for enabling an environment for a toggle](../api/admin/feature-toggles-api-v2.md#enable-env) and specify the environment you'd like to enable.
+
+:::
+
+Use the environments toggles to switch on the environment that you chose above. Depending on the activation strategy you added in the previous step, the toggle should now evaluate to true or false depending on the Unleash context you provide it. To try
+
+![](/img/create-toggle-enable-env.png)
+
+## Optional steps
+
+These optional steps allow you to further configure your feature toggles to add optional payloads, variants for A/B testing, more detailed user targeting and exceptions/overrides.
+
+### Add constraints and segmentation
+
+
+Constraints and segmentation allow you to set filters on your strategies, so that they will only be evaluated for users and applications that match the specified preconditions. Refer to the [strategy constraints](../advanced/strategy-constraints.md "strategy constraints reference documentation") and [segments reference documentation](../reference/segments.mdx) for more information.
+
+To add constraints and segmentation, use the "edit strategy" button for the desired strategy.
+
+![](/img/create-toggle-edit-strategy.png)
+
+
+#### Constraints
+
+:::info
+
+This can either be done when you add a strategy to a toggle or at any point thereafter.
```

Comment:
I think the intent here to reassure the reader that this doesn't have to be done now and isn't set in stone. If that's the case, I would suggest the following (or some variant) instead:

"Constraints aren't fixed and can be changed later to further narrow your audience."

Reply:
It was meant primarily as a way to say that "you need to complete these steps first", but that's a good suggestion nonetheless.